src/BundleAnalyzerPlugin.js

```diff
@@ -35,6 +35,7 @@ class BundleAnalyzerPlugin {
 
     const done = (stats, callback) => {
       callback = callback || (() => {});
+      const bundleDir = stats.compilation.getPath(stats.compilation.outputOptions.path);
```

Comment:
I have no idea if this `getPath` function is a public API from webpack. And neither do I know if `stats.compilation.outputOptions.path` is OK to use here.